x-pack/plugins/endpoint/public/applications/endpoint/store/policy_list/middleware.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { MiddlewareFactory, PolicyListState } from '../../types';
+
+export const policyListMiddlewareFactory: MiddlewareFactory<PolicyListState> = coreStart => {
+  return ({ getState, dispatch }) => next => async action => {
+    next(action);
+
+    if (
+      (action.type === 'userNavigatedToPage' && action.payload === 'policyListPage') ||
+      action.type === 'userPaginatedPolicyListTable'
+    ) {
+      // load data from API
+      // Refactor tracked via: https://github.com/elastic/endpoint-app-team/issues/150
+      setTimeout(() => {
```

Comment:
Why `setTimeout` here?

Reply:
I was trying emulate an API call - so needed a slight delay in order for the Unit Test cases to behave as expected (once we have a real API call in place).


Also - this might be good for me to recap our conversation in Slack here:

I was not sure if I should include a `fake_data` file along with my PR into `master` as a temporary solution until we integrate with the Ingest API. After our discussion, it seems that is ok to do and Resolver is already doing it.  Adding some fake data will allow the team to view the page with data being populated.

So given that - I will add my local face data to the PR (I already had it, just did not include it 😃 )

Reply:
Actually - now I'm thinking you are really saying "why did you not use `await on sleep()`